### ISSUE TITLE
Kertaus - osa 2.1 ensimmäisen tehtävän vaihtoehto tulkinnanvarainen

### ISSUE THREAD

**Issue by TiJoLa** [rage/ohjelmointi-22#27]
Tehtävä:Valitse seuraavista vaihtoehdoista ne, jotka ovat Pythonin syntaksin mukaisia. 
Eikö vaihtoehto "if luku==false:" ole myös syntaksin mukainen, jos on määritellyt muuttujan nimeltään false? Se ei ole kuitenkaan oikeiden vaihtoehtojen joukossa. Ilmeisesti tässä on haettu sitä, että False pitäisi kirjoittaa isolla, mutta estääkö Python nimeämästä false nimistä muuttujaa?

**Comment by bluelhf** [comment]
Ei, ja sen lisäksi lauseketta käyttäessä Python ei anna edes syntaksivirhettä, vaan nimivirheen, joka johtuu siitä, ettei `false` ole määritelty. Tässä demonstraatio, jossa vaihtoehto on täysin validi syntaksiltaan:
![Syntaksia käytetään, ja se toimii moitteetta.](https://user-images.githubusercontent.com/52505120/188076531-08df2dcb-6c01-4815-877e-66acc0b47145.png)
